src/main/scala/edu/ie3/simona/model/participant2/ParticipantModelInit.scala

```diff
@@ -13,6 +13,10 @@ import edu.ie3.simona.agent.participant.data.Data.{
   PrimaryData,
   PrimaryDataExtra,
 }
+import edu.ie3.simona.agent.participant.statedata.ParticipantStateData.{
+  InputModelContainer,
+  WithHeatInputContainer,
```

Comment:
I think this import is unnecessary at the moment.

Reply:
Yes, thats true. Good catch